UnityProject/Assets/Scripts/Core/Lighting System/Light2D/CustomSprite.cs

```diff
@@ -119,7 +119,7 @@ protected virtual void OnEnable()
 	        {
 		        if (Material == null)
 		        {
-			        Material = Resources.GetBuiltinResource<Material>("Sprites-Default.mat");
+			        Material = Resources.GetBuiltinResource<Material>("Default Light Material.mat");
```

Comment:
why the change?

Reply:
it also complaining about not being able to find them but this doesn't work as well

Reply:
maybe its the right resource type?

Reply:
idk